upgrade/manual_ncn_upgrade.md

```diff
@@ -0,0 +1,256 @@
+
+   ```bash
+   source /etc/cray/upgrade/csm/myenv
+   ```
+
+1. (`ncn-m002#`) Copy artifacts from `ncn-m001`.
+
+    > A later stage of the upgrade expects the `docs-csm` and `libcsm` RPMs to be located at `/root/` on `ncn-m002`;
+    > that is why this command copies them there.
+
+   - Install `csi` and `docs-csm`.
+
+       ```bash
+       scp ncn-m001:/root/csm_upgrade.pre_m001_reboot_artifacts.*.tgz /root
+       zypper --plus-repo="/etc/cray/upgrade/csm/csm-${CSM_RELEASE}/tarball/csm-${CSM_RELEASE}/rpm/cray/csm/sle-$(awk -F= '/VERSION=/{gsub(/["-]/, "") ; print tolower($NF)}' /etc/os-release)" --no-gpg-checks install -y cray-site-init
+       scp ncn-m001:/root/*.noarch.rpm /root/
+       rpm -Uvh --force /root/docs-csm-latest.noarch.rpm
+       ```
+
+   - Install `libcsm`.
+
+       > ***NOTE*** Since `libcsm` depends on versions of Python relative to what is included in the SLES service packs,
+       > then in the event that `ncn-m002` is running a newer SLES distro a new `libcsm` must be downloaded. This will
+       > often be the case when jumping to a new CSM minor version (e.g. CSM 1.3 to CSM 1.4).
+       > e.g. if `ncn-m001` is running SLES15SP3, and `ncn-m002` is running SLES15SP4 then the SLES15SP4 `libcsm` is needed.
+       > Follow the [Check for latest documentation](../update_product_stream/README.md#check-for-latest-documentation)
+       > guide again, but from `ncn-m002`.
+
+       ```bash
+       rpm -Uvh --force /root/libcsm-latest.noarch.rpm
+       ```
+
+1. (`ncn-m002#`) Upgrade `ncn-m001`.
+
+   > **`NOTE`** If Kubernetes encryption has been enabled via the [Kubernetes Encryption Documentation](../operations/kubernetes/encryption/README.md),
+    then backup the `/etc/cray/kubernetes/encryption` directory on the master node before upgrading and restore the directory after the node has been upgraded.
+
+   ```bash
+   /usr/share/doc/csm/upgrade/scripts/upgrade/ncn-upgrade-master-nodes.sh ncn-m001
+   ```
+
+   > **`NOTE`** The `root` user password for the node may need to be reset after it is rebooted.
+   Additionally, the `/etc/cray/kubernetes/encryption` directory should be restored if it was backed up.
+   Once it is restored, the `kube-apiserver` on the rebuilt node should be restarted.
+   See [Kubernetes `kube-apiserver` Failing](../troubleshooting/kubernetes/Kubernetes_Kube_apiserver_failing.md) for details on how to restart the `kube-apiserver`.
+
+### Set the image ID and CFS configuration manually
+
+(`ncn-m001#`) Set `XNAME` to the xname of the node that is being upgraded.
+
+```bash
+XNAME=<node_xname>
+```
+
+#### Set the image ID in BSS
+
+1. (`ncn-m001#`) Set `IMS_IMAGE_ID` to the image ID that should be upgraded to.
+
+    ```bash
+    IMS_IMAGE_ID=<image_id>
+    ```
+
+1. (`ncn-m001#`) Set the image ID in BSS.
+
+    ```bash
+    /usr/share/doc/csm/scripts/operations/node_management/assign-ncn-images.sh -p $IMS_IMAGE_ID $XNAME
+    ```
+
+#### Set the CFS configuration in CFS
```

Comment:
If you get rid of the level-2 header at the top of the page, then this can shift down one level.

```suggestion
### Set the CFS configuration in CFS
```